src/lib/blockies/makeBlockiesUrl.ts

```diff
@@ -0,0 +1,49 @@
+import type { BlockiesOptions } from './blockies'
+import { buildBlockiesOptions, createImageData } from './blockies'
+
+const imgData: Record<string, string> = Object.create(null)
+
+export const makeBlockiesUrl = (address: string, size = 8, caseSensitive = false, scale = 10) => {
```

Comment:
do you mind if we wrap this in a lodash `memoize`? she gonna be expensive and this is pure.